modules/apigee_edge_teams/src/Form/TeamContextSwitcherForm.php

```diff
@@ -79,7 +103,13 @@ public function getFormId() {
   /**
    * {@inheritdoc}
    */
-  public function buildForm(array $form, FormStateInterface $form_state, array $teams = []) {
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    // Get the teams for the user.
+    if (!($team_ids = $this->teamMembershipManager->getTeams($this->currentUser->getEmail()))) {
+      return NULL;
```

Comment:
Form builder should always return a (render) array.

Reply:
What should we display if the user has no team?

Reply:
This was a question that I also asked yesterday, I think we should wait for @cnovak 's answer here. I think the empty option list with only "Select team" should be a sane default as the "Teams" link is also always available in the user menu.